pom.xml

```diff
@@ -478,6 +478,20 @@ THE SOFTWARE.
                     <property name="illegalPkgs" value="com.google.common.base, com.google.common.escape, com.google.common.eventbus, com.google.common.graph, com.google.common.hash, com.google.common.html, com.google.common.io, com.google.common.math, com.google.common.net, com.google.common.primitives, com.google.common.reflect, com.google.common.xml, com.google.thirdparty" />
                   </module>
                   <module name="RedundantImport" />
+                  <!--
+                    Javadoc Comments: https://checkstyle.sourceforge.io/config_javadoc.html
+                  -->
+                  <module name="AtclauseOrder">
+                    <property name="tagOrder" value="@param, @return, @serial, @serialData, @serialField, @throws, @version, @deprecated, @exception"/>
```

Comment:
As general order, this looks good:
1. `param`
2. `return`
3. `throws`
4. `deprecated`
👍 

A bit surprised to see `@exception` at the end rather than next to `@throws`, perhaps we could just prohibit that one somehow? And `@version` doesn't look particularly useful either?

Reply:
Sure, I changed this to `@param, @return, @throws, @deprecated`.

Reply:
To clarify, I liked the order as written :) Even without any `@serial*` in the current code, that was fine.